node/popn.go

```diff
@@ -784,6 +795,12 @@ func (nd *node) Get(ctx context.Context, args *GetArgs) {
 		sendErr(err)
 		return
 	}
+
+	// if maxppb is not set during get(), use default node's value
+	if args.MaxPPB == 0 {
```

Comment:
If I want to force a free retrieval with MaxPPB == 0, should I pass a negative number? Wondering how we could differentiate a "use default value" vs "do not pick any offer over 0PPB"

Reply:
Yeah I think negative number seems the easiest way to do it :
-1 -> MaxPPB == 0
0  -> MaxPPB == Default